logger.js

```diff
@@ -95,7 +92,7 @@ module.exports = (externalLogger, logFolder, moduleName = "installer")=> {
     },
     all(evt, detail, pct, table) {
       debug(evt, detail ? detail : "");
-      appendToLog(detail, evt);
+      appendToLog(detail, "event: " + evt);
```

Comment:
@jungeunyoon @santiagonoguez We are keeping this file backwards compatible for Launcher and Player so I would prefer to not add `"event: "`. Keep in mind it will just get logging into -events.log file of a module, so we don't need to see "event: " on every line.

Reply:
@stulees @jungeunyoon OK ! didn't know

Reply:
@Rise-Vision/delivery removed 👍